tests/test_helpers.py

```diff
@@ -158,6 +158,12 @@
      "2020-05-27: lead: ascopatz, sb_firstprojectum, status: proposed\n    Type: mergedpr\n    Title: planning meeting\n    log url: https://docs.google.com/document/d/1YC_wtW5Q\n    Purpose: develop a detailed plan with dates\n    Audience: ascopatz, scopatz, ascopatz\n"
      "2020-05-20: lead: ascopatz, sb_firstprojectum, status: proposed\n    Type: meeting\n    Title: Project lead presentation\n    log url: https://docs.google.com/document/d/1YC_wtW5Q\n    Purpose: lead presents background reading and initial project plan\n    Audience: ascopatz, scopatz, ascopatz\n    Notes:\n      - do background reading\n      - understand math\n"
      ),
+    (["helper", "l_milestones", "--lead", "pliu"],
+     "2021-08-03 (milest): lead: pliu, pl_secondprojectum, Milestone, status: converged\n"
+     ),
+    (["helper", "l_milestones", "--lead", "pliu", "--verbose"],
+     "2021-08-03 (milest): lead: pliu, pl_secondprojectum, status: converged\n    Type: \n    Title: Milestone\n    log url: None\n    Purpose: None\n    Audience: \n    uuid: milestone_uuid_pl2\n"
```

Comment:
This is exactly correct as we want it, except `milest` should be the uuid of the actual milestone that is printed by the test.  See if you can figure out where the code gets it from, and what it is (hint: you may want to open `schemas.py` and look around).  Reach out if you can't.

Reply:
In `schemas.py`, the uuid for all milestones look something like this: `milestone_uuid_pl1`.  That is why I have the `(milest)` bit. If there are exemplar uuids for milestones that look something like `aedes1dsrtesgdw4cdatesd`, I am not seeing those entries. Am I looking at the correct exemplars?